doc/src/manual/performance-tips.md

```diff
@@ -364,45 +370,55 @@ This works fine for `Vector{T}`, but we'd also have to write explicit versions f
 or other abstract types. To prevent such tedium, you can use two parameters in the declaration
 of `MyContainer`:
 
-```julia
-type MyContainer{T, A<:AbstractVector}
-    a::A
-end
-MyContainer(v::AbstractVector) = MyContainer{eltype(v), typeof(v)}(v)
+```jldoctest containers2
+julia> type MyContainer{T, A<:AbstractVector}
+           a::A
+       end
+
+julia> MyContainer(v::AbstractVector) = MyContainer{eltype(v), typeof(v)}(v)
+MyContainer
 
-julia> b = MyContainer(1.3:5);
+julia> b = MyContainer(1:5);
 
 julia> typeof(b)
-MyContainer{Float64,UnitRange{Float64}}
+MyContainer{Int64,UnitRange{Int64}}
 ```
 
 Note the somewhat surprising fact that `T` doesn't appear in the declaration of field `a`, a point
 that we'll return to in a moment. With this approach, one can write functions such as:
 
-```julia
-function myfunc{T<:Integer, A<:AbstractArray}(c::MyContainer{T,A})
-    return c.a[1]+1
-end
-# Note: because we can only define MyContainer for
-# A<:AbstractArray, and any unspecified parameters are arbitrary,
-# the previous could have been written more succinctly as
-#     function myfunc{T<:Integer}(c::MyContainer{T})
+```jldoctest containers2
+julia> function myfunc{T<:Integer, A<:AbstractArray}(c::MyContainer{T,A})
```

Comment:
I think now this has to be `function myfunc(c::MyContainer{T,A}) where T<:Integer where A<:AbstractArray`?